lib/acts_as_taggable_on_mongoid/taggable/list_tags.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ActsAsTaggableOnMongoid
+  module Taggable
+    # This module adds methods for tracking tag definitions within Taggable classes
+    module ListTags
```

Comment:
Added this to manage TagTypeDefinitions.

The code here comes from disparate parts of the source.  Mostly really here:
https://github.com/mbleigh/acts-as-taggable-on/blob/master/lib/acts_as_taggable_on/taggable.rb#L77